### PR TITLE
Create aazad20.json

### PR DESCRIPTION
**Pull request by aazad20**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/aazad20.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Sarvjeet Singh",
+  "bio": "Software Engineer",
+  "tags" : ["Software Engineer","python","react","Javascript","Tensorflow","Data Science"],
+  "socials": [
+    {
+       "icon": "FaLink",
+      "url": "https://www.linkedin.com/in/sarvjeet-singh-6249551b7/",
```
Comment (sital002) [inline data/aazad20.json:8]:
```suggestion
      "url": "https://www.linkedin.com/in/sarvjeet-singh-6249551b7/"
```
The trailing comma isnot required

File: data/aazad20.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      
+    }
+  ]
```
Comment (sital002) [inline data/aazad20.json:11]:
```suggestion
  ],
```

File: data/aazad20.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "links": [
+    {
+      "group": "portfolio",
+      "name": "My portfolio",
+      "url": "https://jeetcodes.netlify.app/",
+      "icon": "FaLink"
+    },
```
Comment (sital002) [inline data/aazad20.json:18]:
```suggestion
    }
```
The trailing comma isnot required

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged.

**Review by sital002 — APPROVED** [review]
Looks good 👍
Welcome to Eddiehub community

**Comment by aazad20** [comment]
Thanks.

**Pull request merged**